src/service/apiClient.js

```diff
@@ -52,4 +52,4 @@ async function request(method, endpoint, data, auth = true) {
   return response.json();
 }
 
-export { login, getPosts, register, createProfile };
+export { login, getPosts, register, createProfile, request, get, post, patch };
```

Comment:
don't export request, get, post, patch -> instead define a new function in this module called `updateUserProfile` that takes in the data to update, the user ID and sends the request for you and returns the response payload, see the `createprofile` for example.

Reply:
Do i have to?  Others on the team also rely on the same exports. They just haven't pr'd it. 
I do understand the benefits of declaring what requests are made in the same file though.